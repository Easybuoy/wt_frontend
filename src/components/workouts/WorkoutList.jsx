import React, { useState, useEffect } from "react";
import WorkoutCard from "./WorkoutCard";
import { Stack, Flex, SimpleGrid } from "@chakra-ui/core";

function WorkoutList() {
  // const [data, setData] = useState()
  // const [isLoading, setIsLoading] =
  return (
    <SimpleGrid columns={3} spacingX={5}>
      <WorkoutCard
        name="Total Body Strength Workout"
        intensity="beginner"
        type="chest"
      />
      <WorkoutCard
        name="Total Body Strength Workout"
        intensity="beginner"
        type="chest"
      />
      <WorkoutCard
        name="Total Body Strength Workout"
        intensity="beginner"
        type="chest"
      />
    </SimpleGrid>
  );
}

export default WorkoutList;
