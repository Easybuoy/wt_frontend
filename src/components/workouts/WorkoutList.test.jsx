import React from "react";
import { shallow } from "enzyme";

import WorkoutList from "./WorkoutList";

describe("<WorkoutList />", () => {
  let wrapper;

  const props = {
    location: { search: jest.mock() },
    client: { mutate: jest.fn(), query: jest.fn() },
    history: { push: jest.fn() }
  };

  beforeEach(() => {
    wrapper = shallow(<WorkoutList {...props} />);
  });

  it("renders the WorkoutList component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
