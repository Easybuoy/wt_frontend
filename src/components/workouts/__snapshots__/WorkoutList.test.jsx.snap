// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WorkoutList /> renders the WorkoutList component correctly 1`] = `
<styled.div>
  <Search
    placeholder="Look for a specific workout"
  />
  <withApollo(Workouts) />
</styled.div>
`;
